javascript/form.js

```diff
@@ -0,0 +1,31 @@
+/*Missing: 
+- Functionaltiy on new Card
+- Minlength function*/
+
+console.clear();
+
+const form = document.querySelector('[data-js="form"]');
+const mainElement = document.querySelector('[data-js="form_main"]');
+
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+  const formData = new FormData(event.target);
+  const data = Object.fromEntries(formData);
+
+  const newCard = document.createElement("article");
+  newCard.classList.add("card");
+  const cardText = question.value;
+  const answerText = answer.value;
+  newCard.innerHTML = `
+  <h2>${cardText}</h2>
```

Comment:
you might want to put the question text into a smaller text container. h2 might be a little huge, if the question is getting longer, it runs out of bounce quite quick.
You could also style the h2 font-size to fix that.